<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>StatisticsPanel</class>
  <widget class="QDialog" name="StatisticsPanel">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>300</width>
        <height>350</height>
      </rect>
    </property>
    <property name="sizePolicy">
      <sizepolicy>
        <hsizetype>5</hsizetype>
        <vsizetype>5</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
    <property name="minimumSize">
      <size>
        <width>300</width>
        <height>300</height>
      </size>
    </property>
    <property name="baseSize">
      <size>
        <width>350</width>
        <height>300</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>Statistics</string>
    </property>
    <property name="windowIcon">
      <iconset resource="resources.qrc" >
      <normaloff>:/images/images/LDViewIcon16.png</normaloff>:/images/images/LDViewIcon16.png</iconset>
    </property>
    <property name="sizeGripEnabled">
      <bool>false</bool>
    </property>
    <layout class="QVBoxLayout">
      <property name="margin">
        <number>11</number>
      </property>
      <property name="spacing">
        <number>4</number>
      </property>
      <item>
        <layout class="QGridLayout">
          <property name="margin">
            <number>0</number>
          </property>
          <property name="spacing">
            <number>4</number>
          </property>
          <item row="0" column="0">
            <widget class="QLabel" name="partsLabel">
              <property name="text">
                <string>Parts:</string>
              </property>
              <property name="alignment">
                <set>Qt::AlignVCenter|Qt::AlignRight</set>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="1" column="0">
            <widget class="QLabel" name="modelsLabel">
              <property name="text">
                <string>Models:</string>
              </property>
              <property name="alignment">
                <set>Qt::AlignVCenter|Qt::AlignRight</set>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="2" column="0">
            <widget class="QLabel" name="trianglesLabel">
              <property name="text">
                <string>Triangles:</string>
              </property>
              <property name="alignment">
                <set>Qt::AlignVCenter|Qt::AlignRight</set>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="3" column="0">
            <widget class="QLabel" name="quadsLabel">
              <property name="text">
                <string>Quads:</string>
              </property>
              <property name="alignment">
                <set>Qt::AlignVCenter|Qt::AlignRight</set>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="4" column="0">
            <widget class="QLabel" name="linesLabel">
              <property name="text">
                <string>Lines:</string>
              </property>
              <property name="alignment">
                <set>Qt::AlignVCenter|Qt::AlignRight</set>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="5" column="0">
            <widget class="QLabel" name="edgeLinesLabel">
              <property name="text">
                <string>Edge Lines:</string>
              </property>
              <property name="alignment">
                <set>Qt::AlignVCenter|Qt::AlignRight</set>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="6" column="0">
            <widget class="QLabel" name="conditionalLinesLabel">
              <property name="text">
                <string>Conditional Lines:</string>
              </property>
              <property name="alignment">
                <set>Qt::AlignVCenter|Qt::AlignRight</set>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="0" column="1">
            <widget class="QLineEdit" name="partsLine">
              <property name="readOnly">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item row="1" column="1">
            <widget class="QLineEdit" name="modelsLine">
              <property name="readOnly">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item row="2" column="1">
            <widget class="QLineEdit" name="trianglesLine">
              <property name="readOnly">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item row="3" column="1">
            <widget class="QLineEdit" name="quadsLine">
              <property name="readOnly">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item row="4" column="1">
            <widget class="QLineEdit" name="linesLine">
              <property name="readOnly">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item row="5" column="1">
            <widget class="QLineEdit" name="edgeLinesLine">
              <property name="readOnly">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item row="6" column="1">
            <widget class="QLineEdit" name="conditionalLinesLine">
              <property name="readOnly">
                <bool>true</bool>
              </property>
            </widget>
          </item>
         <item row="7" column="1">
            <widget class="QPushButton" name="okButton">
              <property name="sizePolicy">
                <sizepolicy>
                  <hsizetype>0</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>OK</string>
              </property>
              <property name="shortcut">
                <string/>
              </property>
              <property name="default">
                <bool>true</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="4" margin="11"/>
  <pixmapfunction></pixmapfunction>
  <resources>
   <include location="resources.qrc" />
  </resources>
</ui>
